contracts/swaps/connectors/SwapsImplSovrynSwap.sol

```diff
@@ -84,10 +84,20 @@ contract SwapsImplSovrynSwap is State, ISwapsImpl {
 
         ISovrynSwapNetwork sovrynSwapNetwork =
             getSovrynSwapNetworkContract(sovrynSwapContractRegistryAddress);
+
+        IERC20[] memory _defaultPathConversion =
+            defaultPathConversion[sourceTokenAddress][destTokenAddress];
+
+        /// will use the defaultPath if it's set, otherwise query from the SovrynSwapNetwork.
         IERC20[] memory path =
-            sovrynSwapNetwork.conversionPath(IERC20(sourceTokenAddress), IERC20(destTokenAddress));
+            _defaultPathConversion.length >= 3
```

Comment:
```suggestion
            _defaultPathConversion.length > 0
```
the same suggestion as in the ProtocolSettings contract

Reply:
i mean https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/434#discussion_r897369137